packages/scene-composer/src/components/Tree/TreeItem.tsx

```diff
@@ -119,6 +121,18 @@ const TreeItem = React.forwardRef<HTMLLIElement, TreeItemProps>(
               variant='inline-icon'
               onClick={expandHandler}
               iconName={`treeview-${expanded ? 'collapse' : 'expand'}`}
+              ariaExpanded={expanded}
+              ariaLabel={`${
+                expanded
+                  ? formatMessage({
+                      defaultMessage: 'Collapse',
+                      description: 'Label for button that triggers a collapsable section',
+                    })
+                  : formatMessage({
+                      defaultMessage: 'Expand',
+                      description: 'Label for button that triggers a expandable section',
+                    })
+              } ${(labelText as any)?.props?.labelText || ''}`}
```

Comment:
Why does labelText need to be casted to `any`? There is nearly always a better way to do this and I implore you to find it. If you need help or anything, let me know!

Reply:
Hi Tracy, I will look into this, I had just copied this from elsewhere in this file without thinking.